### ISSUE TITLE
Consider moving version.pm to DualLife organization

### ISSUE THREAD

**Issue by atoomic** [Perl/version.pm#18]
I was wondering why version.pm is treated differently than other dual life distributions.

Should not the GitHub repository used to publish it belong to https://github.com/Dual-Life instead of Perl?

If so can we consider moving the repo from Perl to Dual-Life?

**Comment by Grinnz** [comment]
It's also a very unique dual life distribution in that it has such a close relation to core. But either way would make sense in my opinion.

**Comment by atoomic** [comment]
Yes for sure it's a special one, and has some files outside of its main repository like `vxs.inc` and `vutil.[ch]`.
But I'm sure some dual life distro are also special. For example `ExtUtils-ParseXS`, `if`, `base`, `threads`... 

I can also live with both options, but as most other dual life distro owned by p5p belongs to the Dual-Life organization,
I think it makes more sense to move it there.

**Comment by haarg** [comment]
https://github.com/Dual-Life/version.pm already exists as a fork of this repo, and it's currently ahead by one commit.

Either org seems like a reasonable home for this, but having both exist is awful. This repo has pull requests and issues, so the fork in Dual-Life should be deleted. Then if desired, this repo can be moved to the Dual-Life org.

**Comment by Leont** [comment]
> https://github.com/Dual-Life/version.pm already exists as a fork of this repo, and it's currently ahead by one commit.

I have deleted that repo.

> Then if desired, this repo can be moved to the Dual-Life org.

I don't have a strong opinion either way on that.
